app/build.gradle.kts

```diff
@@ -130,6 +130,15 @@ kotlin {
     }
 }
 
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        if (project.hasProperty("localSdk")) {
```

Comment:
I don't think `gradle.properties` is the appropriate file to add the `localSdk` flag. I think `user.properties` is a better candidate since it's already required (to set `gitHubToken`), and it's ignored so there's no chance of accidentally tracking the `localSdk` flag.

I would also suggest only conditionally adding `mavenLocal()` when the `localSdk` flag is set to true.

Reply:
Changed to `user.properties`.

Reply:
@Hinton Will that change to `user.properties` be reflected in https://sdk.contributing-docs.pages.dev/getting-started/sdk/#linking-sdk-to-clients ?

Reply:
@brian-livefront Just updated it.

Reply:
Great, thanks @Hinton !